python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -829,6 +889,29 @@ def _get_subprocess_kwargs(self):
 
         return kwargs
 
+    def _get_task_parent_entity_type(self, task_id):
+        """
+        Gets the Task entity's parent entity type.
+        """
+        cache = self.WSS_KEY_CACHE[self._wss_key]
+
+        if "task_parent_types" in cache and task_id in cache["task_parent_types"]:
```

Comment:
TASK_PARENT_TYPES = "task_parent_types" at the top of the file would be appreciated. This is non standard enough that it is worth constantifying.